bazel/linux/defs.bzl

```diff
@@ -991,3 +991,31 @@ def kernel_test(name, kernel_image, module, rootfs_image = None, test_dir_cfg =
     if rootfs_image:
         cfg = mconfig(cfg, rootfs_image = rootfs_image)
     name_runner = mcreate_rule(name, runner, "", runner_cfg, kwargs, cfg)
+
+def _bundle(ctx):
+    out = []
+
+    for dep in ctx.attr.deps:
+        for f in dep.default_runfiles.files.to_list():
+            out.append(f)
+
+    d = files_to_dir(
+        ctx,
+        ctx.attr.name + "-root",
+        out,
+    )
+
+    return [
+        DefaultInfo(files = depset([d]))
```

Comment:
here, in addition to DefaultInfo, you must return a KernelRuntimeInfo for the bundle to be usable with the other qemu/uml rules I added so far.

To return a KernelRuntimeInfo, you're half way there: you are using files_to_dir to shove all the files in a directory (good!), the part that is missing is generating an init script to run? to execute the bundle.

Suggestions to get there:
- each dep above contains "an executable" to run, the way I'd access it is something like `dep[DefaultInfo]'.files_to_run.executable` and then `.path` (or `.short_path`, never get it right).
- I'd use `ctx.actions.declare_file` and `ctx.actions.write` to generate a shell script (our init) that just runs each executable one after the other with some helpful debug messages, and propagates the exit status. 
- I'd pass the `file object` returned by `ctx.actions.declare_file` to `files_to_dir` to make sure the init is actually copied into the output dir.